### PR TITLE
chore: Fix some TODOs

### PR DESCRIPTION
**Pull request by dgusakov**
Interfaces were not changed

### CODE REVIEW

**Review by madlabman — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

File: src/CSModule.sol [hunk 1]
```diff
@@ -157,7 +154,7 @@ contract CSModule is
     error NodeOperatorDoesNotExist();
     error SenderIsNotEligible();
     error InvalidVetKeysPointer();
-    error StuckKeysHigherThanTotalDeposited();
+    error StuckKeysHigherThanTotalDepositedMinusTotalExited();
```
Comment (skhomuti) [inline src/CSModule.sol:157]: Higher than active?

File: src/CSModule.sol [hunk 2]
```diff
@@ -302,15 +300,17 @@ contract CSModule is
         // Due to new bonded keys nonce update is required and normalize queue is required
         _updateDepositableValidatorsCount({
             nodeOperatorId: nodeOperatorId,
-            doIncrementNonce: true,
-            doNormalizeQueue: true
+            incrementNonceIfUpdated: true,
+            normilizeQueueIfUpdated: true
```
Comment (skhomuti) [inline src/CSModule.sol:304]: Normalize

File: src/CSModule.sol [hunk 3]
```diff
@@ -302,15 +300,17 @@ contract CSModule is
...
         });
     }
 
     /// @notice Adds a new node operator with stETH bond
+    /// @notice Due to the stETH rouding issue make sure to make approval or sign permit with extra 10 wei to avoid revert
```
Comment (skhomuti) [inline src/CSModule.sol:309]: Rounding - typo

File: src/CSModule.sol [hunk 4]
```diff
@@ -302,15 +300,17 @@ contract CSModule is
...
     /// @param keysCount Count of signing keys
     /// @param publicKeys Public keys to submit
-    /// @param signatures Signatures of (deposit_message, domain) tuples
+    /// @param signatures Signatures of (deposit_message_root, domain) tuples.
+    ///                   https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#signingdata
```
Comment (skhomuti) [inline src/CSModule.sol:313]: To be sure the link wouldn't be stale you can pin some ref in it

**Pull request merged**